PyU4V/tests/unit_tests/test_migrate.py

```diff
@@ -0,0 +1,960 @@
+# Copyright (c) 2019 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+OpenStack test_migrate.py.
+
+Test file for migrate.py and migrate_utils.py
+"""
+import platform
+
+from . import pyu4v_common_data as pcd  # noqa: H304
+from . import pyu4v_fakes as pf  # noqa: H304
+from PyU4V import provisioning
+from PyU4V import rest_requests
+from PyU4V.tools.openstack import migrate_utils
+from PyU4V import univmax_conn
+from PyU4V.utils import exception
+
+import mock
+import testtools
+
+
+class TestMigrate(testtools.TestCase):
+    """Test cases for migrate script."""
+
+    def setUp(self):
+        """Set up the test class."""
+        super(TestMigrate, self).setUp()
+        self.data = pcd.CommonData()
+        self.conf_file, self.conf_dir = (
+            pf.FakeConfigFile.create_fake_config_file())
+        univmax_conn.file_path = self.conf_file
+        with mock.patch.object(
+                rest_requests.RestRequests, 'establish_rest_session',
+                return_value=pf.FakeRequestsSession()):
+            conn = univmax_conn.U4VConn()
+        self.utils = migrate_utils.MigrateUtils(conn)
+
+    def test_get_masking_view_component_dict_old(self):
+        """Test for get_masking_view_component_dict."""
```

Comment:
Same docstring used across multiple tests in this file, is it possible to append a descriptive unique identifier to the end for these comments so that each test has a unique docstring?